.github/workflows/codeql.yml

```diff
@@ -31,9 +31,16 @@ jobs:
         languages: 'java'
         # Use only 'java' to analyze code written in Java, Kotlin or both
 
-    # Autobuild attempts to build any compiled languages (C/C++, C#, Go, Java, or Swift).
-    - name: Autobuild
-      uses: github/codeql-action/autobuild@v2
+    - name: Set up JDK 19
+      uses: actions/setup-java@v1
+      with:
+        java-version: 19
+
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+
+    - name: Build with Gradle
+      run: ./gradlew build -x test
```

Comment:
@bell-pa do you think this is an appropriate way of building the code for scanning? I've disabled the tests here as they're running on CI and we just need the code to be built to scan for vulnerabilities, but let me know if there's a better approach.

Running the build this way was also resulting in some failing tests, not sure why 🤷 

Anyway, this seems to work!